nucypher/utilities/json_metrics_export.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+from prometheus_client.utils import floatToGoString
+from twisted.web.resource import Resource
+from prometheus_client.registry import REGISTRY
+import json
+
+
+class JSONMetricsResource(Resource):
+    """
+    Twisted ``Resource`` that serves metrics in JSON.
+    """
+    isLeaf = True
+
+    def __init__(self, registry=REGISTRY):
+        self.registry = registry
+
+    def render_GET(self, request):
+        request.setHeader(b'Content-Type', "text/json")
+        return generate_latest_json(self.registry)
+
+
+def generate_latest_json(registry):
+    '''Returns the metrics from the registry in latest JSON format as a string.'''
+    output = {}
+    for metric in registry.collect():
+        try:
+            for s in metric.samples:
+                sample_labels = {}
+                if s.labels:
+                    sample_labels = {k: v.replace('\\', r'\\').replace('\n', r'\n').replace('"', r'\"') for k, v in
```

Comment:
That's odd - what is the cause of the formatting Issue here?

Reply:
My bad! Forgot to remove it after some copypaste.
Fixed.